### DISCUSSION TITLE
MSSQL and Authentication for the Web Project

### DISCUSSION THREAD

**Discussion by theCJMan** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#78] [Q&A] [answered]
Good day
I am about to embark on a journey to add MS SQL to the web project and Authentication for users.
Is this something you want in the Sample web?
1. It then is becoming less of a sample and more of a solution
2. The requirements may then become more strenuous because the user may have to have access to a MS SQL server
3. More nuGet packages
4. Screens for reading messages (webhooks).
5. etc.

If this is not the direction you want to take this code then I might have to break from adding more code to your project.

Please let me know, Feel free to contact me via WhatsApp: +27790995412
thanx

**Comment by gabrieldwight** [comment, marked as answer]
Hi, the sample web project was meant to be kept simple to only focus on how to use the WhatsApp cloud API functions. Adding a database connection to the sample won't be a good idea since other users may not use MS SQL server while they use other database providers.

**Reply by theCJMan** [reply]
100%
I have now completed the webhook implementation ion the Sample app and started with some Data stuff, but you can ignore that.
Will create a Pull now..
Pls do contact me on whatsapp..
